DEVELOPMENT.md

```diff
@@ -21,6 +21,39 @@ The pull request ref goes in brackets at the end of the subject.
 Generally we also follow [good commit message hygene][tpope].
 
 
+## Work coordination
+
+Work on the app and proxy can happen in parallel. However for this to work the
+teams working on both code bases have to agree on a common API. The definitive
+source-of-truth for this API is the proxy source code.
+
+To change or extend the API, first open an issue on GitHub, discuss what is
+needed and then propose proxy code changes that implement the API.
+
+We don't keep around a copy of a SDL schema file as they tend to get outdated
+quickly and it is way easier to explore the API via introspection. See the next
+section for more information on that.
+
+
+### GraphiQL explorer
+
+The introspective and self-docummenting nature of GraphQL allows us to easily
+explore our API by querying the backend. For better developer ergonomics we
+include a [static version][gs] of [GraphiQL-explorer][ge]. To see it in action:
+
+1. Start the app in development mode: `(cd app && yarn start)`
+2. Open GraphiQL in your browser via the CLI: `(cd app && yarn graphiql)` or
```

Comment:
This command produces the following error on my Arch machine: `/bin/sh: open: command not found`

Reply:
Hm, just tried with `bash` and `zsh` both worked, which shell are you using?

Reply:
Using `fish` on ARch.

Reply:
Alright, should be fixed now.